drivers/display/uc81xx.c

```diff
@@ -425,8 +417,7 @@ static int uc81xx_write(const struct device *dev, const uint16_t x, const uint16
 	__ASSERT(!(desc->width % UC81XX_PIXELS_PER_BYTE),
 		 "Buffer width not multiple of %d", UC81XX_PIXELS_PER_BYTE);
 
-	if ((y_end_idx > (config->height - 1)) ||
-	    (x_end_idx > (config->width - 1))) {
+	if ((y_end_idx > (config->height - 1)) || (x_end_idx > (config->width - 1))) {
```

Comment:
It'd be nice if you could keep the same formatting as as before to keep the diff as small as possible. That keeps the git annotate log clean and avoids merge conflicts in the future.

Reply:
ah sorry about that, fixed :)